pkg/network/setup/netstat_test.go

```diff
@@ -61,28 +62,34 @@ var _ = Describe("netstat", func() {
 			),
 		).To(Succeed())
 
+		setup.Vmi.Status.Interfaces = []v1.VirtualMachineInstanceNetworkInterface{
```

Comment:
I do not understand this addition. Why is it needed?

In general, I do not understand why you had to add this podIface thing into the existing tests.
You could have tested it separately, right?
As there is no `network-status` I would expect everything to just continue to pass.

For the cases where the information is available, you can add the relevant tests per the conditions the production code presents.

Reply:
This code was removed.
Unit tests for this file are currently missing.